### DISCUSSION TITLE
Translation tools

### DISCUSSION THREAD

**Discussion by redtide** [agateau/colorpick#13] [Ideas]
LXQt has a project called [lxqt-build-tools](https://github.com/lxqt/lxqt-build-tools). In that tool suite there are some CMake helper modules made for translations, which include also .desktop files translations in YAML format.

I would like to use them in my projects but unfortunately they are not supporting yet Qt6, so I made a "fork" called [Qtilitools](https://github.com/qtilities/qtilitools/). I use them in my projects as external dependency but I guess in this case might be better, eventually, to include them in a cmake/ folder.

Would it be Ok to add these in the repo? I can make a single PR with them and adding my Italian translations.

**Comment by agateau** [comment]
I am not sure there is a need for such complexity: It's only one single string to translate, and it's unlikely to change once it's set. I'd rather ask translators to provide the string when a new translation is added and be done with it.

**Reply by redtide** [reply]
> It's only one single string to translate

Not really, there are GenericName, Comment and optionally Name in the desktop file, though this can be done directly over the original desktop file if using GitHub, the YAML method turns handy when working on a portal like Weblate.

Other (at least) 5 strings are in the copy to clipboard menu and other 5 (included the one you mentioned) in the ContrastPreview, but yes, once made they hardly change.

Anyway, I posted #17.
